routes/jobs.js

```diff
@@ -0,0 +1,15 @@
+// routes/routes.js
+const express = require('express');
+const { create, list } = require('../controllers/jobsController');
+const jobsRoutes = express.Router();
+
+const authMiddleware = require('../middlewares/authMiddleware');
+
+
+// Create a new job
+jobsRoutes.post('/create', authMiddleware, create);
```

Comment:
I've added a authPlusActiveMiddleware, please check if that could be better here